ci-operator/step-registry/openshift/e2e/openstack/nfv-experimental/openshift-e2e-openstack-nfv-experimental-workflow.yaml

```diff
@@ -4,8 +4,6 @@ workflow:
     pre:
       - chain: ipi-openstack-pre
       - ref: openstack-provision-performanceprofile
-      - ref: openstack-provision-vfio-noiommu
```

Comment:
> this is being removed because we're loading from the kernelargs now right?
Yes, once Sebastian's PR to support vhostuser has landed, we can remove `openstack-provision-vfio-noiommu` because the SR-IOV network operator will take care of it.

>  if yes, do we plan to remove from the other workflows later?
Absolutely! But first we use the experimental workflow to test things on demand, before changing the "stable" periodic jobs that use the "legacy" manual steps.

Does it make sense?

Reply:
sure. I thought this step `openstack-provision-vfio-noiommu` could already be excluded since this was merged k8snetworkplumbingwg/sriov-network-operator#257